README.md

```diff
@@ -2,25 +2,34 @@
 
 # Neurobagel API
     
-[![Tests](https://img.shields.io/github/actions/workflow/status/neurobagel/api/test.yaml?color=BDB76B&label=test&style=flat-square)](https://github.com/neurobagel/api/actions/workflows/test.yaml)
-[![Coverage](https://img.shields.io/coverallsCoverage/github/neurobagel/api?style=flat-square&color=8FBC8F)](https://coveralls.io/github/neurobagel/api)
-[![Python](https://img.shields.io/badge/python-3.10-4682B4?style=flat-square)](https://www.python.org/)
-[![License](https://img.shields.io/github/license/neurobagel/api?color=CD5C5C&style=flat-square)](LICENSE)
+[![Tests](https://img.shields.io/github/actions/workflow/status/neurobagel/api/test.yaml?color=BDB76B&label=test&style=flat)](https://github.com/neurobagel/api/actions/workflows/test.yaml)
+[![codecov](https://codecov.io/gh/neurobagel/api/graph/badge.svg?token=ZEOGQFFZMJ)](https://codecov.io/gh/neurobagel/api)
+[![Python](https://img.shields.io/badge/python-3.10-4682B4?style=flat)](https://www.python.org/)
```

Comment:
How about adding a badge for the latest-built Docker image version as well? see https://github.com/neurobagel/bagel-cli/blob/5694ebee520699a8cb03b1f12116ccfedce5de4d/README.md?plain=1#L6 for example